tools/docker/hub/Dockerfile

```diff
@@ -13,5 +13,3 @@ RUN set -xe; \
 # Stage 2
 FROM ${BASE_IMAGE}
 COPY --from=0 /work/implementations/elixir/ockam/ockam_hub/_build/prod/rel/ockam_hub /opt/ockam_hub
-ENTRYPOINT ["/opt/ockam_hub/bin/ockam_hub"]
-CMD ["start"]
```

Comment:
I think we need to keep this as the default entrypoint and command ... the azure container instance will run as long as the default command blocks. If we don't have this then the entrypoint of the base image will have to used some how to define a blocking command.

Reply:
done